pkgs/development/libraries/audio/lv2/default.nix

```diff
@@ -1,45 +1,73 @@
 { stdenv
 , lib
 , fetchurl
+, asciidoc
+, codespell
+, doxygen
+, gtk2
+, libsamplerate
 , libsndfile
 , pkg-config
 , python3
-, wafHook
+, meson
+, ninja
 , pipewire
+, serd
+, sord
+, gitUpdater
 }:
 
 stdenv.mkDerivation rec {
   pname = "lv2";
-  version = "1.18.2";
+  version = "1.18.10";
 
-  outputs = [ "out" "dev" ];
+  outputs = [ "out" "dev" "doc" ];
 
   src = fetchurl {
-    url = "https://lv2plug.in/spec/${pname}-${version}.tar.bz2";
-    sha256 = "sha256-TokfvHRMBYVb6136gugisUkX3Wbpj4K4Iw29HHqy4F4=";
+    url = "https://lv2plug.in/spec/${pname}-${version}.tar.xz";
+    hash = "sha256-eMUbzyG1Tli7Yymsy7Ta4Dsu15tSD5oB5zS9neUwlT8=";
   };
 
   nativeBuildInputs = [
+    asciidoc
+    codespell
```

Comment:
This does not really make sense to have in package. It will check spelling issues in the code.

Reply:
AFAIU it's pulled in by default when tests are ran: https://gitlab.com/lv2/lv2/-/blob/master/test/meson.build#L9. What would be the better option? Patch the test out, disable tests? At a glance the dependency does not look large.

Reply:
Probably best to adjust it upstream. Dependencies meant for development that are not useful for consumers should not be coupled to tests. Either there should be a separate flag, or just use `required: false`.

Reply:
Disabled docs for now.